packages/core/src/view/handler/CellEditorHandler.ts

```diff
@@ -312,7 +312,7 @@ class CellEditorHandler implements GraphPlugin {
       this.textarea.style.minHeight = '1em';
     }
 
-    this.textarea.style.position = 'relative';
+    this.textarea.style.position = 'absolute';
```

Comment:
**question:** Have you checked this change outside the StoryBook stories to see if there's any change in behavior? In the js-example or ts-example applications provided in this repository, for example?
In mxGraph, the position was calculated dynamically: https://github.com/jgraph/mxgraph/blob/v4.2.2/javascript/src/js/view/mxCellEditor.js#L281
It seems to have been removed during migration and I don't know why. Probably because of the name of the function and its comments, even though it seems to have had an impact on modern browsers (and not only in IE 8).

Here, we could at least introduce a way to set the value with a new property and let people decide what they want to use as a position.
I haven't thought this through any further, so feel free to suggest alternatives or contradict the proposal.l 😸

Reply:
Yes you are right, I also tested with `ts-example` and other standalone applications.
I reproduce the problem for both the vertices and the edges.